nbs/tsdataset.ipynb

```diff
@@ -54,13 +54,15 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #36.                # Add Available mask efficiently (without adding column to df)
</pre><p>I think this logic can be extracted to the base class</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>